### DISCUSSION TITLE
How can i add TOC/LOF/LOT

### DISCUSSION THREAD

**Discussion by cigit-zgy** [quarto-dev/quarto-cli#7916] [Q&A]
### Description

How can i add TOC/LOF/LOT into the pdfbookmark

Nowadays, I’m writing my doctoral thesis, I’d like to add TOC/LOF/LOT into the pdfbookmark so that I can find these quickly.

Thanks for your kind help.

**Comment by mcanouil** [comment]
I don't understand what you want.
TOC is already part of PDF bookmark.
<img width="1624" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/5dc987d1-915b-49cd-bb5d-f48abeaa19c7">

Nota that the the list of figures/tables, I suggest you searched on the internet in more dedicated places about LaTeX.
